docs/evaluate/production-features/temporal-cloud/high-availability.mdx

```diff
@@ -0,0 +1,119 @@
+---
+id: high-availability
+title: High availability - Temporal Cloud production feature
+description: Add push-button high availability for your Temporal Service to ensure continuous reliable operation.
+sidebar_label: High availability
+tags:
+  - production-deployment
+  - production-features
+  - temporal-cloud
+  - high-availability
+keywords:
+  - availability
+  - explanation
+  - failover
+  - high-availability
+  - multi-region
+  - multi-region namespace
+  - namespaces
+  - temporal-cloud
+  - term
+---
+
+:::tip Support, stability, and dependency info
+- Multi-region Namespaces are in [Public Preview](/evaluate/release-stages#public-preview) for Temporal Cloud.
+
+:::
+
+Temporal Cloud offers disaster-tolerant deployment for workloads with stringent availability requirements.
+With the multi-region feature enabled, Temporal Cloud automates [failover](/glossary#failover] and synchronizes data between Namespace regions.
+
+This page introduces Temporal Cloud patterns that support your workload's availability requirements.
+
+## Multi-region Namespaces
+
+Multi-region Namespaces provide failover capabilities to mitigate service outages due to regional failures.
+They reduce risk and minimize operational disruption.
+This feature seamlessly shif​​ts Workflow execution between regions to maintain service availability.
+
+Your Clients use a single logical Namespace with a single endpoint that operates in two physical regions: one active and one standby.
+As Workflows progress in the active region, history events asynchronously replicate to the standby region.
+Data replication ensures both regions are in sync so the standby is ready to take over when needed.
+
+Should an incident or outage occur in the active region, Temporal Cloud initiates a "failover" to the standby region.
+During a failover, the roles of the active and standby regions reverse.
+The standby takes over as the primary region.
+
+### Advantages of multi-region Namespaces {#multi-region-advantages}
```

Comment:
Please check - We capitalize Temporal product terms, even in titles. So should it be...

```suggestion
### Advantages of Multi-region Namespaces {#multi-region-advantages}
```

Reply:
The standard is lower-case, and matching with single-region (which also takes a hyphen when mentioned, but there is no SRN)